### ISSUE TITLE
Proxy Feature?

### ISSUE THREAD

**Issue by Coto16** [chinoogawa/instaBrute#16]
Hello, I was wondering how you could use: https://github.com/chinoogawa/instaBrute/tree/feature/proxy-support

**Comment by aleperno** [comment]
### The throughout explanation 
In the example available you should have a TOR running which will be used as a proxy.
A positional parameter has been added to the script, therefore to enable the proxy within the script ```-p``` or ```--proxy``` should in the arguments when called.

Selenium webdriver has the options to set proxy settings. To store the settings I used a ```yaml``` configuration file (Json-like) which holds both arguments to be passed to the webdriver ```set_preference``` method. The config is read, for now only the first entry is used. For every key_value, it calls the method setting the preferences. Then updates them.

Bear in mind:
 - Currently only the first entry is used.
 - For different proxies, the keys are different. You should refer to webdriver docs.

And more fundamentally, this is **NOT** a fully featured script, this should serve as a PoC and a ground base for anyone to improve / build upon it.

### TL;DR
Have a TOR browser running (which will be used as proxy), then run the script as allways but adding `-p` or `--proxy` as argument.
